src/controllers/orderController.js

```diff
@@ -92,6 +128,17 @@ const deleteOrder = async (req, res) => {
   await Order.findByIdAndDelete(orderId); // or more simply can just call delete on the documnent we've already fetched:  await order.remove()
 
   res.status(StatusCodes.OK).json({ msg: 'Success! Order was deleted' });
+  /*
+  #swagger.summary = 'Delete an order by id'
+  #swagger.description = '**ROLE REQUIRED:** user'
```

Comment:
This endpoint will also work for admins

```suggestion
  #swagger.description = '**ROLE REQUIRED:** user or admin'
```